galasa-ui/src/components/TokenDeleteModal.tsx

```diff
@@ -9,23 +9,15 @@ import { useRef, useState } from 'react';
 import { TextInput } from '@carbon/react';
 import { InlineNotification } from '@carbon/react';
 import { Loading,Modal} from "@carbon/react"
+import Token from '@/utils/interfaces/Token';
 
-export default function TokenRequestModal({ tokens, selectedTokens, handleUpdateTokens, handleUpdateDeleteModalState }: { tokens: any, selectedTokens: any, handleUpdateTokens: any, handleUpdateDeleteModalState:any }) {
+export default function TokenRequestModal({ tokens, selectedTokens, deleteTokenFromSet, updateDeleteModalState }: { tokens: Set<Token>, selectedTokens: Set<string>, deleteTokenFromSet: Function, updateDeleteModalState:Function }) {
```

Comment:
Minor - this could be cleaned up by creating a TokenRequestModalProps interface, making this:

```typescript
export default function TokenRequestModal({ tokens, selectedTokens, deleteTokenFromSet, updateDeleteModalState }: TokenRequestModalProps)
```

Same applies when you find yourself adding a lot of props to a component.